udev/udev-block-add-change

```diff
@@ -49,21 +49,20 @@ refresh_another() {
     env -i PATH=$PATH $launch_env $0
 }
 
-# communicate with xenstored through socket in dom0
-# trying to access xenstore before xenstored is started, hang forever (in
-# non-killable state), so better fail ('-s' in VM if /proc/xen isn't mounted
-# yet) than hang dom0 boot
-if [ ! -r /proc/xen/capabilities ] || grep -q control_d /proc/xen/capabilities; then
```

Comment:
\o/
This was one of the last usage of legacy `/proc/xen` xenfs (causing [various problems](https://github.com/QubesOS/qubes-issues/issues/2540))!